react/features/jwt/TokenData.js

```diff
@@ -0,0 +1,59 @@
+/* @flow */
+
+import jwtDecode from 'jwt-decode';
+
+/**
+ * Represent the data parsed from the JWT token
+ */
+export default class TokenData {
+
+    /**
+     * Object representing callee data.
+     */
+    callee: Object;
+
+    /**
+     * Object representing caller data.
+     */
+    caller: Object;
+
+    /**
+     * Contains group. Taken from the token payload.
+     */
+    group: string;
+
+    /**
+     * Issuer claim identifies the principal that issued the JWT.
+     */
+    issuer: string;
+
+    /**
+     * Raw jwt token itself.
+     */
+    jwt: string;
+
+    /**
+     * Contains server. Taken from the token payload.
+     */
+    server: string;
+
+    /**
+     * Creates instance of token data.
+     *
+     * @param {string} jwt - The JWT token.
+     */
+    constructor(jwt: string) {
+        const payload = jwtDecode(jwt);
+
+        if (!payload || !payload.context) {
+            return;
+        }
+
+        this.callee = payload.context.callee;
+        this.caller = payload.context.user;
+        this.group = payload.context.group;
+        this.issuer = payload.issuer;
```

Comment:
That does not look right! The (reserved) issuer claim is called `iss`. The name `iss` appears to be used in the source code in master but you have switched to `issuer` here.